dockernet/pkill.sh

```diff
@@ -0,0 +1,3 @@
+pkill -f "docker-compose .*simd.* logs" | true
+pkill -f "/bin/bash.*create_logs.sh" | true
```

Comment:
Can we change it so that we only get logs when we run with special make command and have a seperated command for running with and without make commands?

Reply:
Yes, I 'll add new make command with debug flag when we want get logs.